rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -183,6 +183,65 @@ def _verify_oidc_config_id(oidc_config_id, rosa_cmnd, my_path):
     return False
 
 
+def _verify_provision_shard(ocm_cmnd, shard_id):
+    logging.info('Verifing Shard ID: %s' % shard_id)
+    ocm_command = [ocm_cmnd, "get", "/api/clusters_mgmt/v1/provision_shards/" + shard_id]
```

Comment:
We should confirm the status of the shard is either ready or maintenance

Reply:
add verification of status